Containerfile

```diff
@@ -0,0 +1,22 @@
+FROM registry.fedoraproject.org/fedora:36
```

Comment:
Should we try `fedora-minimal:36`?

https://fedoramagazine.org/build-smaller-containers/